python_modules/libraries/create-dagster/create_dagster/templates/PROJECT_NAME_PLACEHOLDER/src/PROJECT_NAME_PLACEHOLDER/definitions.py.jinja

```diff
@@ -5,4 +5,4 @@ from dagster import definitions, load_from_defs_folder
 
 @definitions
 def defs():
-    return load_from_defs_folder(project_root=Path(__file__).parent.parent.parent)
+    return load_from_defs_folder(project_root=Path(__file__).parent)
```

Comment:
this should be `path_within_project` instead of `project_root`, worth the rename on public api `load_from_defs_folder`?